rhsso/1.3.5/keycloakoperator.1.3.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,195 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: keycloakoperator.1.3.5
+  namespace: placeholder
+  annotations:
```

Comment:
Only changes from the operator registry manifests were these annotations @philbrookes